src/itrim.cpp

```diff
@@ -122,17 +121,17 @@ usage() {
         "Options are:",
         " -M  admin - admin user uses this option to trim other users files",
         " -n  replNum - the replica number of the replica to be deleted",
-        " -N  numCopies - minimum number of most current copies to keep. The default",
+        " -N  numCopies - minimum number of good replicas to keep. The default",
         "     value is 2.",
         " -r  recursive - trim the whole subtree",
         " -S  srcResource - specifies the resource of the replica to be deleted.",
-        "     If specified, only copies stored in this resource will be candidate",
+        "     If specified, only replicas stored in this resource will be candidate",
         "     for the deletion.",
```

Comment:
The first line seems weird. It's worded as if the option is used to target a **single** replica, yet your updated line mentions **replicas** (which is correct).

Reply:
agreed, will update.